teradatalabs/mapr52-hive/files/wardenTracker.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+hname=$(hostname)
```

Comment:
@ArturGajowy  can you review Kerberbos MapR which is new also the test changes we did 
I will also look into the old PR comments and fix together